src/moscot/plotting/_utils.py

```diff
@@ -20,6 +20,8 @@
 from moscot._constants._constants import AggregationMode
 from moscot.problems.base._compound_problem import K
 
+_N = 200
```

Comment:
Is there a need for this constant?

Reply:
yuo're right, we can take the default value 256